tests/strictequal.test.js

```diff
@@ -0,0 +1,24 @@
+
+var describe = require('mocha').describe
+var it = require('mocha').it
+var expect = require('chai').expect
+
+var strictEqual = require('../strictequal')
+
+
+describe('strictEqual',() => {
+    it ('return true when types are the same or different', () => {
```

Comment:
There should be four tests checking on each type combination.  Watch the explainer video when I put it up and this will al become clearer.  You had the right idea just couldn't get it to work together.  And remember if you don't understand something reach out for help do not just use something you find on the internet that you do not know anything about.  You won't learn that way.  It is great that you are looking things up but its about learning the fundamentals that you will be building on.